operator/config/rbac/kustomization.yaml

```diff
@@ -4,6 +4,6 @@ resources:
 # if your manager will use a service account that exists at
 # runtime. Be sure to update RoleBinding and ClusterRoleBinding
 # subjects if changing service account names.
-- service_account.yaml
```

Comment:
can we move this out of operator and into a top level folder like develop

Reply:
this will be need for dev, I will move them `/develop` folder with next PR